discord.bat

```diff
@@ -1,3 +1,41 @@
+@echo off
+chcp 65001 >nul
+:: 65001 - UTF-8
+
+:: Path check
+set scriptPath=%~dp0
+set "path_no_spaces=%scriptPath: =%"
+if not "%scriptPath%"=="%path_no_spaces%" (
+    echo Путь содержит пробелы. 
+    echo Пожалуйста, переместите скрипт в директорию без пробелов.
+    pause
+    exit /b
+)
+
+:: Cyrillic check
+setlocal enabledelayedexpansion
+set "cyrillic_found=0"
+for /l %%i in (0,1,127) do (
+    set "char=!scriptPath:~%%i,1!"
+    for %%c in (А Б В Г Д Е Ё Ж З И Й К Л М Н О П Р С Т У Ф Х Ц Ч Ш Щ Ъ Ы Ь Э Ю Я а б в г д е ё ж з и й к л м н о п р с т у ф х ц ч ш щ ъ ы ь э ю я) do (
+        if "!char!"=="%%c" set "cyrillic_found=1"
+    )
+)

```

Comment:
That's significantly increasing run time. On most devices script works with Cyrillic symbols in path. Let's not check it at all. Same for all others scripts